### PR TITLE
Fixed zk-tip message spacing

### PR DESCRIPTION
**Pull request by T-Damer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "eth-rpc-errors": "^4.0.3",
     "ethers": "^5.6.6",
     "fortmatic": "^2.3.0",
-    "preact": "https://github.com/preactjs/preact",
+    "preact": "^10.7.2",
```
Comment (T-Damer) [inline package.json:38]: `preact` failed to install for me, so I reinstalled it, looks like they've merged the `svg` types PR :)
https://github.com/preactjs/preact

Comment (T-Damer) [reply package.json:38]: It also failed the GH Actions

File: yarn.lock [hunk 2]
```diff
@@ -16612,3 +16612,4 @@ __metadata:
   checksum: f77b3d8d00310def622123df93d4ee654fc6a0096182af8bd60679ddcdfb3474c56c6c7190817c84a2785648cdee9d721c0154eb45698c62176c322fb46fc700
   languageName: node
   linkType: hard
+
```
Comment (backmeupplz) [inline yarn.lock:16615]: This is unnecessary change

File: src/components/ProofsCard.tsx [hunk 3]
```diff
@@ -35,9 +36,11 @@ const proofCardZKButtonContainer = classnames(
 )
 function ZkProofSavedMessage() {
   return (
-    <AccentText small primary color="text-primary">
-      Created ZK proofs are saved in the browser even if you switch wallets.
-    </AccentText>
+    <div className={margin('mt-2')}>
```
Comment (backmeupplz) [inline src/components/ProofsCard.tsx:39]: Are you sure you shouldn't use `space` here instead?

**Review by backmeupplz — CHANGES_REQUESTED** [review]

**Review by backmeupplz — APPROVED** [review]

File: src/components/ProofsCard.tsx [hunk 4]
```diff
@@ -25,7 +25,11 @@ import useProofAddressesAvailableToCreate from 'hooks/useProofAddressesAvailable
 const titleContainer = space('space-y-2')
 const innerScrollableBlock = space('space-y-2')
 const proofContainer = space('space-y-6')
-const proofBlockSpacing = space('space-y-2')
+const proofBlockSpacing = classnames(
```
Comment (backmeupplz) [inline src/components/ProofsCard.tsx:28]: This is no longer just spacing, make sure to rename the variable

**Review by backmeupplz — CHANGES_REQUESTED** [review]
